### DISCUSSION TITLE
Want to increase the .RPU framerate to 59.940 fps from default 23.976

### DISCUSSION THREAD

**Discussion by chesley96** [quietvoid/dovi_tool#283] [Q&A] [answered]
Hello, if anyone can help I would greatly appreciate it! I am demuxing the DV's or HDR10Plus's respective metadata from the source, then using TensorRT RIFE to increase the framerate of the video stream to 59.940 fps, losing HDR obviosly.  Then I inject the .RPU/json back into the interpolated file with 2.5 times the original frames. In doing so the RPU frames get smushed into the first half of the file causing massive flickering.  I have edited the scripts accordingly to CHGFPS to 59.940 to match the total frames but the result is the same. 
I believe I need to change the framerate and frame time of the RPU.bin or HDR10Plus.json stream to 59.940 fps but do not know how or if it is possible. 
![HDR10_ HDR10+ _ HDR10+ Plot](https://github.com/quietvoid/dovi_tool/assets/15711209/c07a2298-ebfa-40ca-a983-d8409eb40cf6)

Video
Format                                   : HEVC
Format/Info                              : High Efficiency Video Coding
Format profile                           : Main 10@L6.1@Main
HDR format                               : SMPTE ST 2094 App 4, Version 1, HDR10+ Profile B compatible
Width                                    : 3 840 pixels
Height                                   : 2 144 pixels
Display aspect ratio                     : 16:9
Frame rate                               : 59.940 (60000/1001) FPS
Color space                              : YUV
Chroma subsampling                       : 4:2:0
Bit depth                                : 10 bits
Color range                              : Limited
Color primaries                          : BT.2020
Transfer characteristics                 : PQ
Matrix coefficients                      : BT.2020 non-constant

**Comment by quietvoid** [comment, marked as answer]
This isn't really a supported usecase.
You'd essentially need to duplicate the metadata in the same way.

If you use a frameserver, it would technically be possible to dump the RPU for every frame.

**Reply by chesley96** [reply]
Thank you for getting back to me so quickly! I'm looking into that now and was unaware of a frameserver. Thanks again!

**Comment by kinger9527** [comment]
Torrent Galaxy is no longer available

**Comment by 60Fpsfan** [comment]
@chesley96 can you help me with something can you send your mail please
unfortunately tgx go down so

**Comment by card0z** [comment]
@chesley96 Can you please give me a way to contact you. I would be really grateful. Having a hard time with my svp transcodes and Dolby Vision Content.
